README.md

```diff
@@ -7,47 +7,39 @@
 [![Quality Score](https://img.shields.io/scrutinizer/g/xparse/RecursivePagination.svg?style=flat-square)](https://scrutinizer-ci.com/g/xparse/RecursivePagination)
 [![Total Downloads](https://img.shields.io/packagist/dt/xparse/recursive-pagination.svg?style=flat-square)](https://packagist.org/packages/xparse/recursive-pagination)
 
-Recursive Pagination allows you to parse through website pages. You need to set xPath to pagination area and xPath to pages that have such pagination. 
+Recursive Pagination allows you to parse through website pages. 
+You need to pass link from where to start and set xPath or CSS expression to pagination links. 
 
-## Install
+## Installation
 
-Via Composer
+You can install the package via Composer
 
 ``` bash
 $ composer require xparse/recursive-pagination
 ```
 
-## Usage
-
-```php
-  $parser = new \Xparse\Parser\Parser();
-  $linksArrayPath = [
-      "//a[@class='categoryitem']/@href",     // path to pages you want to scrape
-      "//td[@class='pagination']//a/@href"    // path to pagination area
-    ];
+## Basic Usage
+``` php
+  use Xparse\Parser\Parser;
+  use Xparse\RecursivePagination\RecursivePagination;
   
-  $paginator = new RecursivePagination($parser, $linksArrayPath);
-  $paginator->addToQueue('http://www.example.com/first/page/to/parse.html');
+  # init Parser
```

Comment:
Try to find all links to the repositories in github. Our query will be `xparse`. 
With recursive pagination we can traverse all pagination links and fetch page from github.com